.github/workflows/upgrade-python-requirements.yml

```diff
@@ -16,9 +16,9 @@ jobs:
     uses: openedx/.github/.github/workflows/upgrade-python-requirements.yml@master
     with:
       branch: ${{ github.event.inputs.branch }}
-      team_reviewers: "arbi-bom"
-      email_address: arbi-bom@edx.org
-      send_success_notification: false
+      team_reviewers: "wg-maintenance-edx-platform"
+      # email_address: example@example.com
```

Comment:
Let’s make this orbi-bom@edx.org, and they’ll support as best they can for now. Possibly they can do some leg work and earn their way to deeper support.

Reply:
Happy to have them get e-mails, but I want to make sure the github reviewing team can merge these so that works for me if that sounds good to you.